src/nft/components/bag/profile/ListingModal.tsx

```diff
@@ -139,98 +173,106 @@ const ListingModal = () => {
   const clickStartListingFlow = () => {
     resetAllRows()
     allCollectionsApproved ? signListings() : startListingFlow()
+    !allCollectionsApproved && sendAnalyticsEvent(EventName.NFT_SELL_START_LISTING, { ...startListingEventProperties })
```

Comment:
I think you might want this before the flow gets started as this event may appear out of order, ie after all signatures approved

Reply:
Or alternatively, add this event to the start of `startListingFlow`